Automattic/abacus
# Add backend errors to Wizard

**This PR adds shows backend error messages to the Experimenter in the Experiment Wizard.**
- Adds ServerErrorAlert component


<img width="1005" alt="Screen Shot 2021-01-08 at 3 49 14 pm" src="https://user-images.githubusercontent.com/971886/103988425-1525b080-51c9-11eb-9c24-34a1220eefc5.png">


Resolves: #415 

## How has this been tested?


To get a Server Error:
1. Spin up a production data Abacus
2. Clone an explat_test experiment
3. Copy the experiment_name from the original
4. (Update the start/end dates)
5. Try submit, see same error in screenshot above

To get a successful submit:
1. Create a normal experiment






- Automated tests that cover added/changed functionality
- Manual testing with production data (locally)
